src/main/java/org/alliancegenome/curation_api/jobs/JobScheduler.java

```diff
@@ -146,7 +146,7 @@ public void runMassIndexerEverything() {
 
 		if(reindexSchedulingEnabled){
 			Log.info("Scheduled mass reindexing initiated.");
-			bulkLoadDAO.reindexEverything(1000, 10000, 0, 4, 7200, 1);
+			bulkLoadDAO.reindexEverything(1000, 10000, 0, 4, 14400, 1);
```

Comment:
Should this be 28800 like the rest?

Reply:
No, the idea would be that for the automated reindexing, we use a shorter timeout (4h here) which we deem the maximum "acceptable" duration, which would cause the automated reindexing to fail when going over that time, giving us an indication we need to create a ticket to fix it (and potentially hold a deployment until a fix is in place). The other timeouts are the defaults that go to swagger, which should be longer such that developers triggering the indexing don't need to retry several times increasing the TransactionTimeout each time (8h in my current proposal).

Reply:
I've now set the timeout for automated reindexing back to 2h, and the default for swagger UI to 4h.